homework/index1.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="UTF-8">
+        <title>HUDA ALISMAIL</title>
+        <link rel="stylesheet" href ="styles/styles.css">
+
+    <body>
+        <div class="name-section">          
+            <h1> Joe Johnson</h1>
+            <span class="border"></span>
+      
+
+      <div class="pro-photo">
+      <img src ="styles/profi.jpg">
+      </div>
+
+    
+      <header class="pages">
+      
+          <a href ="index.html">About Page</a>
+          <a href ="index1.html">Portfolio</a>
+
+      </header>
+      </div>
+           
+          
+
+      
+
+            <div>
+                <div class="head-section"> 
```

Comment:
be consistant with indentation to make the code easier to read 

here is a resource you can use for more information
https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html